src/lib/containers/GenericCard.tsx

```diff
@@ -33,6 +34,7 @@ export type GenericCardSchema = {
   imageFileHandleColumnName?:string
   secondaryLabels?: any[]
   link?: string
+  dataType?: string
```

Comment:
Is there an example of using this new prop, in the demo app or in a unit test?

Reply:
Yes, just added.